src/components/Toggle/Toggle.stories.mdx

```diff
@@ -0,0 +1,180 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { HStack, VStack, Grid } from "@chakra-ui/react";
+import Heading from "../Heading/Heading";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+import Toggle from "./Toggle";
+import { ToggleSizes } from "./ToggleSizes";
+
+<Meta
+  title={getCategory("Toggle")}
+  component={Toggle}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=31581%3A302",
+    },
+    jest: ["Toggle.test.tsx"],
+  }}
+  argTypes={{
+    className: { table: { disable: true } },
+  }}
+/>
+
+# Toggle
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.25.3`   |
+| Latest            | `0.25.3`   |
+
+<Description of={Toggle} />
```

Comment:
This doesn't do anything. You need to add esdoc style comment above the `Toggle` component function declaration for it to render here.